sidebar.jsx

```diff
@@ -141,25 +156,28 @@ class Sidebar extends React.Component {
           <button className="sb-smallButton">
             Make an Offer
           </button>
-          <button className="sb-smallButton" id="watchButton" onClick={this.watchProduct.bind(this)}>
+          <button className="sb-smallButton sb-floatRight" id="watchButton" onClick={this.watchProduct.bind(this)}>

```

Comment:
This line exceeds 80 characters. Could you use the following pattern to overcome this?
```
<button
  [property]
  [property]
>
```